.vscode/ltex.dictionary.en-US.txt

```diff
@@ -0,0 +1,2 @@
+Zeebe
+Tasklist
```

Comment:
I'm thinking this .vscode file should be added to the .gitignore. WDYT? Something like this - https://github.com/github/gitignore/blob/main/Global/VisualStudioCode.gitignore although we may just need to add 
`.vscode/*`

Reply:
Agreed. I just popped in the latter.